src/components/AuthProvider/AuthProvider.js

```diff
@@ -0,0 +1,23 @@
+import { Auth0Provider } from "@auth0/auth0-react";
+
+const AUTH0_USER = "dev-xsron5cxoe45bn57.us.auth0.com"
+const AUTH0_KEY = "zmgQ6uO27tIDiDuJjGC4ZF0M3l0OySlv"
+// const AUTH0_USER = process.env.DOMAIN
+// const AUTH0_KEY = process.env.CLIENT_ID
+// const AUTH0_SECRET = process.env.CLIENT_SECRET
```

Comment:
Save the auth0 data in the .env file and share this file throw Slack with the dev team

Reply:
> 

Okay